### PR TITLE
Fix unit test that is breaking because of hardcoded cookie expiration date

### PR DESCRIPTION
**Pull request by ErikBooijCB**

### CODE REVIEW

**Comment by ErikBooijCB** [comment]
There was a hardcoded expiration date for a cookie in this test, which was far enough in the future when the test was written, but that day has actually come by now. I've updated it to always use the next day instead of using a hardcoded value. This test breaks all current inspections.

**Comment by dbu** [comment]
2019-02-02 just because :-D

File: tests/Cookie/CookieJarTest.php [hunk 1]
```diff
@@ -366,7 +369,7 @@ public function testCanConvertToAndLoadFromArray()
     public function testAddsCookiesWithEmptyPathFromResponse()
     {
         $response = new Response(200, array(
-            'Set-Cookie' => "fpc=foobar; expires=Fri, 02-Mar-2019 02:17:40 GMT; path=;"
+            'Set-Cookie' => "fpc=foobar; expires=Fri, {$this->futureExpirationDate()}; path=;"
```
Comment (ranvis) [inline tests/Cookie/CookieJarTest.php]: Though not critical for strtotime(), day `Fri, ` is left?

Comment (ErikBooijCB) [reply tests/Cookie/CookieJarTest.php]: @ranvis Thanks, nice spot, it was sloppy coding on my part, fixed now.

**Comment by sagikazarmark** [comment]
Thanks!

**Comment by wirwolf** [comment]
Hello @ErikBooijCB, i create a fork of this repo and release this marge request into the new version(6.4.0).

Namespace compatibility is 100%, and you can use my changes if you patch composer.json in your projects
* replace `"guzzlehttp/guzzle": "^6.3"` to `"someblackmagic/guzzle": "^6.4"` 
* add `"guzzlehttp/guzzle": "6.3.*"` into replace block

Fork link: https://github.com/SomeBlackMagic/guzzle/

**Pull request merged**